app/(main)/(screens)/(profile)/textbox-page.web.tsx

```diff
@@ -0,0 +1,109 @@
+import React, { useRef, useState } from "react";
+import { useTheme } from "@react-navigation/native";
+import { Text, View } from "@/components/theme/Themed";
+import { Button } from "react-native-paper";
+import { CreateCampaignstylesFn } from "@/styles/profile/TextBox.styles";
+import { useLocalSearchParams, useRouter } from "expo-router";
+import AppLayout from "@/layouts/app-layout";
+import Colors from "@/constants/Colors";
+import { RichEditor, RichToolbar } from "react-native-pell-rich-editor";
```

Comment:
import { RichEditor, RichToolbar } from "react-native-pell-rich-editor";
import ReactQuill from "react-quill";

These are working on both native and web, right?
Please make sure nothing breaks.